apache/superset
# feat(config): Add none force-refresh option for auto refresh

### SUMMARY

Add alternative option for issue #16944 
If people think this should be default, I may just remove from config and simply change force -> fetch

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

**Before:**
https://user-images.githubusercontent.com/43336767/136601879-c3ea7040-5944-43b6-bf05-15d7f29ef22a.mov

When DASHBOARD_FORCE_AUTO_REFRESH is True, chart will query every time when it auto-refreshes.

**After:**
https://user-images.githubusercontent.com/43336767/136601893-a876f406-6190-4d90-bffb-e78e4eaa8661.mov

When DASHBOARD_FORCE_AUTO_REFRESH is False, chart will use cache if it exist during auto-refresh.

### TESTING INSTRUCTIONS

Since this PR requires to change superset-ui to change, you can test by following step instead.
1. Open dashboard with several charts with cache of certain period (Say 300s)
2. Check that chart will not use cache at all (like before video) during auto-refresh
3. Change fetchCharts(xx, True, xx, xx) to  fetchCharts(xx, false, xx, xx)
4. Check that chart use cache during auto-refresh

### ADDITIONAL INFORMATION
